compact/definitions.xml

```diff
@@ -424,7 +424,7 @@ Examples:
 
     <constant name="CentralTrackingRegion_rmax"   value="715.0*mm" />
     <constant name="CentralTrackingRegionP_zmax"  value="1730.0*mm" />
-    <constant name="CentralTrackingRegionN_zmax"  value="1474.7*mm" />
+    <constant name="CentralTrackingRegionN_zmax"  value="1186.0*mm" />
```

Comment:
Help needed here, to make the z-positioning consistent with the mRICH (outside->in, rather than inside->out)